packages/assistant/src/RPA/Assistant/library.py

```diff
@@ -0,0 +1,991 @@
+    ) -> None:
+        """Add a centered heading text element
+
+        :param heading: The text content for the heading
+        :param size:    The size of the heading
+
+        Supported ``size`` values are Small, Medium, and Large. By default uses
+        the value Medium.
+
+        Example:
+
+        .. code-block:: robotframework
+
+            Add heading     User information  size=Large
+            Add heading     Location          size=Small
+            Add text input  address           label=User address
+            Run dialog
+        """
+        if not isinstance(size, Size):
+            size = Size(size)
+
+        if size == Size.Small:
+            self._client.add_element(element=Text(heading, style="headlineSmall"))
+        elif size == Size.Medium:
+            self._client.add_element(element=Text(heading, style="headlineMedium"))
+        elif size == Size.Large:
+            self._client.add_element(element=Text(heading, style="headlineLarge"))
+
+    @keyword("Add text")
+    def add_text(
+        self,
+        text: str,
+        size: Size = Size.Medium,
+    ) -> None:
+        """Add a text paragraph element, for larger bodies of text
+
+        :param text: The text content for the paragraph
+        :param size: The size of the text
+
+        Supported ``size`` values are Small, Medium, and Large. By default uses
+        the value Medium.
+
+        Example:
+
+        .. code-block:: robotframework
+
+            Add heading   An error occurred
+            Add text      There was an error while requesting user information
+            Add text      ${error}   size=Small
+            Run dialog
+        """
+        if not isinstance(size, Size):
+            size = Size(size)
+
+        if size == Size.Small:
+            self._client.add_element(element=Text(text, style="bodySmall"))
+        elif size == Size.Medium:
+            self._client.add_element(element=Text(text, style="bodyMedium"))
+        elif size == Size.Large:
+            self._client.add_element(element=Text(text, style="bodyLarge"))
+
+    @keyword("Add link")
+    def add_link(
+        self,
+        url: str,
+        label: Optional[str] = None,
+    ) -> None:
+        """Add an external URL link element
+
+        :param url:   The URL for the link
+        :param label: A custom label text for the link
+
+        Adds a clickable link element, which opens the user's default
+        browser to the given ``url``. Optionally a ``label`` can be given
+        which is shown as the link text, instead of the raw URL.
+
+        Example:
+
+        .. code-block:: robotframework
+
+            Add heading    An error occurred
+            Add text       See link for documentation
+            Add link       https://robocorp.com/docs    label=Troubleshooting
+            Run dialog
+        """
+        if not label:
+            label = url
+        self._client.add_element(Markdown(f"[{label}]({url})"))
```

Comment:
Would this benefit from escaping the content?

Reply:
not sure yet, I'll check this and also what library we use already for escaping (if any) as Flet doesn't include it